src/lib/trap.ts

```diff
@@ -52,3 +80,14 @@ export function trap<T>(
     })
   return true
 }
+
+/**
+ * Trap an error, suppressing it from the user.  Trapping is the opposite of
+ * propagating an error.  We should propagate errors in low-level functions and
+ * trap at the top level.
+ */
+export function trapSuppressed<T>(
+  value: ExcludeErr<T> | Error
+): value is Error {
+  return trap(value, { suppress: true })
+}
```

Comment:
Why is this function necessary at all when they can just call with the necessary parameter?

Reply:
I decided not to use it, but I thought others might want it. I can delete it.

Reply:
To me it's dead code but up to you